docs/src/introduction/getting-started.md

```diff
@@ -1 +1,131 @@
+
+```shell
+miden-client tx new mint <regular-account-ID> <faucet-account-id> 1000
+```
+
+This will execute, prove and submit a transaction that mints assets to the node. The account that executes this transaction will be the faucet as was defined in the node's configuration file. In this case, it is minting `1000` fungible tokens to `<regular-account-ID-A>`.
+
+This will add a transaction and an output note (containing the minted asset) to the local store in order to track their lifecycles. You can display them by running `miden-client tx list` and `miden-client input-notes list` respectively. If you do so, you will notice that they do not show a `commit height` even though they were submitted to the operator. This is because our local view of the network has not yet been updated. After updating it with a `sync`, you should see the height at which the transaction and the note containing the asset were committed. This will allow us to prove transactions that make use of this note, as we can compute valid proofs that state that the note exists in the blockchain.
+
+## Step 4: Consuming the note
+
+After creating the note with the minted asset, the regular account can now consume it and add the tokens to its vault. You can do this the following way:
+
+```bash
+miden-client tx new consume-note <regular-account-ID-A> <input-note-ID>
+```
+
+This will consume the input note ID, which you can get by listing them as explained in the previous step. You will now be able to see the asset in the account's vault by running:
+
+```bash
+miden-client account show <regular-account-ID-A> -v
+```
+
+## Step 5: Transferring assets between accounts
+
+Some of the tokens we minted can now be transferred to our second regular account. To do so, you can run:
+
+```shell
+miden-client sync # Make sure we have an updated view of the state
+miden-client tx new p2id <regular-account-ID-A> <regular-account-ID-B> <faucet-account-ID> 50 # Transfers 50 tokens to account ID B
+```
+
+This will generate a Pay-to-ID (`P2ID`) note containing 50 assets, transferred from one regular account to the other. If we sync, we can now make use of the note and consume it for the receiving account:
+
+```shell
+miden-client sync # Make sure we have an updated view of the state
+miden-client tx new consume-note <regular-account-ID-B> # Consume the note
+```
+
+That's it! You will now be able to see `950` fungible tokens in the first regular account, and `50` tokens in the remaining regular account:
+
+```shell
+miden-client account show <regular-account-ID-B> -v # Show account B's vault assets (50 fungible tokens)
+miden-client account show <regular-account-ID-A> -v # Show account A's vault assets (950 fungible tokens)
+```
+
+### Clearing the state
+
+All state is maintained in `store.sqlite3`, located in the same directory where the client binary is. In case it needs to be cleared, the file can be deleted; it will later be created again when any command is executed.
```

Comment:
The store is now located on the directory as per what is defined in the `miden-client.toml`